rockspec/apisix-master-0.rockspec

```diff
@@ -32,7 +32,7 @@ description = {
 
 dependencies = {
     "lua-resty-ctxdump = 0.1-0",
-    "lua-resty-dns-client = 5.2.0",
+    "lua-resty-dns-client = 5.2.3",
```

Comment:
Maybe we can use the latest release directly?

Reply:
Well, let me first look at the compatibility